unsubscribe-automation/src/server.ts

```diff
@@ -0,0 +1,54 @@
+import fastify from "fastify";
+import cors from "@fastify/cors";
+import { autoUnsubscribe } from "./main";
+import { z } from "zod";
+
+const server = fastify();
+
+// Register CORS
+server.register(cors, {
+  origin: process.env.CORS_ORIGIN || "http://localhost:3000",
```

Comment:
Can remove: `|| "http://localhost:3000",`
If you want to set it as an env var to be localhost:3000 you can.
In production this may not be set at all.

Reply:
Done